### PR TITLE
run Janus Graph integration tests against scylla

### PR DESCRIPTION
**Pull request by abvgedeika**
https://github.com/scylladb/scylla/issues/2497
>=2 nodes in cluster, 1 loader

### CODE REVIEW

**Comment by slivne** [comment]
why do we need this via sct - can't we do it with ccm ?

**Comment by slivne** [comment]
the same that we do with driver tests

**Comment by duarten** [comment]
We should have a version to test integration with the thrift driver as well (I think they still have it).

**Comment by abvgedeika** [comment]
@slivne when I run in dtest I see many failures due to timeout. so, results much better with the real cluster )

**Comment by slivne** [comment]
It would help to understand what the timeouts are about ... are the
integration tests using long duration / a lot of data etc.

File: janus_graph_test.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    def test_janus_graph_regression(self):
+        """
+        Test steps:
+
+        1. git clone https://github.com/JanusGraph/janusgraph.git && cd janusgraph
+        2. mvn clean install -DskipTests -pl janusgraph-cql -am
+        3. mvn verify -pl janusgraph-cql -Dstorage.hostname=DB_IP -Dtest=CQLGraphTest
+        """
+        self.loaders.nodes[0].remoter.run('sudo yum install -y git maven')
+        self.loaders.nodes[0].remoter.run('git clone https://github.com/JanusGraph/janusgraph.git && cd janusgraph')
```
Comment (amoskong) [inline janus_graph_test.py]: unused cd command.

File: janus_graph_test.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        self.loaders.nodes[0].remoter.run('cd janusgraph && mvn clean install -DskipTests -pl janusgraph-cql -am')
+        self.loaders.nodes[0].remoter.run(
+            'cd janusgraph && mvn verify -pl janusgraph-cql -Dstorage.hostname={0} -Dtest=CQLGraphTest'.format(
+                self.db_cluster.nodes[0].private_ip_address), ignore_status=True)
+        dest = os.path.join(self.logdir, 'janusgraph-cql')
+        os.mkdir(dest)
+        self.loaders.nodes[0].remoter.receive_files(src='/home/centos/janusgraph/janusgraph-cql/target/',
```
Comment (amoskong) [inline janus_graph_test.py]: Can we replace the hardcode path with `~/janusgraph/janusgraph-cql/target/`?

**Comment by abvgedeika** [comment]
@slivne the tests are not long duration and not a lot of data only slight increasing of timeout before ttl expiration verification fix the tests https://github.com/abvgedeika/janusgraph/commit/aebdcb5ce117ca478a88ff5a5e50a189f4f47a64 

as sjudeng already mentioned they do not plan to increase it in their tests because the scenarios are used in all their storage backends:
 The tests are common across all JanusGraph storage backends (Cassandra CQL/Thrift/Astyanax/Embedded, HBase and BerkeleyDB). I'm not sure if sub-second expiration is required or just tested. At this point I'm just assessing whether ScyllaDB can be dropped in seamlessly for our Cassandra backend. Again the initial results are great, so kudos to your team! I think this issue would be a pretty trivial caveat for us to make regarding using JanusGraph with ScyllaDB, though of course ideally there would be no caveats.

File: janus_graph_test.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    # https://github.com/scylladb/scylla/issues/2497
+    def test_janus_graph_regression(self):
+        """
+        Test steps:
+
+        1. git clone https://github.com/JanusGraph/janusgraph.git && cd janusgraph
+        2. mvn clean install -DskipTests -pl janusgraph-cql -am
+        3. mvn verify -pl janusgraph-cql -Dstorage.hostname=DB_IP -Dtest=CQLGraphTest
+        """
+        self.loaders.nodes[0].remoter.run('sudo yum install -y git maven')
```
Comment (lmr) [inline janus_graph_test.py]: Idea: assign `self.loaders.nodes[0]` to a variable. Saves typing.

File: janus_graph_test.py (hunk 2 above)
Comment (lmr) [reply janus_graph_test.py]: @amoskong I don't think `~` resolves to `/home/centos` in this context.

**Pull request merged**